cvinfer/io.py

```diff
@@ -218,6 +225,15 @@ def _next(self):
                     self.close()
                     break
 
+    def get_frame_count(self):
+        return self.frame_count
+
+    def get_total_frame_count(self):
```

Comment:
you can get this info from `self.metadata["frame_count"]` without having to open the file handler again